lib/kitchen/directions/bake_chapter_introductions/v2.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterIntroductions
+  class V2
+    def bake(book:, chapter_objectives_strategy:)
+      book.chapters.each do |chapter|
+        introduction_page = chapter.introduction_page
+
+        introduction_page.search("div[data-type='description']").trash
+        introduction_page.search("div[data-type='abstract']").trash
+
+        title = introduction_page.title.cut
+        title.name = 'h2'
+        Kitchen::Directions::MoveTitleTextIntoSpan.v1(title: title)
+
+        chapter_objectives_html =
+          Kitchen::Directions::BakeChapterIntroductions.bake_chapter_objectives(
+            chapter: chapter,
+            chapter_objectives_strategy: chapter_objectives_strategy
+          )
+
+        intro_content = introduction_page.search('[data-type="note"].chapter-objectives').cut
+        extra_content = introduction_page.search(
+          '> :not([data-type="metadata"]):not(.splash):not(.has-splash)'
+        ).cut
+
+        introduction_page.append(child:
+          <<~HTML
+            <div class="intro-body">
+              #{chapter_objectives_html}
+              #{intro_content.paste}
+              <div class="intro-text">
+                #{title.paste}
+                #{extra_content.paste}
+              </div>
+            </div>
+          HTML
+        )
+      end
+
+      Kitchen::Directions::BakeChapterIntroductions.v1_update_selectors(book)
+    end
```

Comment:
So V2 has no way to bake chapter outlines? Am I reading that right?

Reply:
oop, since we ran into books that don't need it, I didn't include it in v2, I was thinking that when book recipes get refactored and start using v2, we could add another strategy to call `bake_chapter_outline`, since the books that do bake a chapter outline are using v1. Do you think it's necessary to do it in this PR?

Reply:
If it's depreciating v1, then v2 needs to cover the functionality of v1 :T 

If it's quick, you could add that in, but we just want to move forward, we could un-depreciate v1, and then make a new card to make v2 have a strategy for baking chapter outlines as well